src/Autocomplete/src/Form/WrappedEntityTypeAutocompleter.php

```diff
@@ -168,4 +169,9 @@ private function getEntityMetadata(): EntityMetadata
 
         return $this->entityMetadata;
     }
+
+    public function setOptions(array $options): void
+    {
+        $this->options = $options;
```

Comment:
To be extra safe, if `$this->form` is set, throw an exception. It means the user called this too late.